runtime/tests/tests.rs

```diff
@@ -307,53 +286,110 @@ fn staking_genesis_config_works() {
 }
 
 #[test]
-fn staking_reward_should_work() {
+fn staking_inflation_and_reward_should_work() {
```

Comment:
As per the offline discussion: since this test doesn't do anything special outside the staking module it should be part of the staking test suite, it's not a full runtime integration test so shouldn't be here

Reply:
Yep will be addressed in a follow up PR

Reply:
Yeah, we could move it and also the transaction payment test into `multi_token_test` in staking module